packages/pigeon/lib/objc_generator.dart

```diff
@@ -1444,7 +1444,8 @@ String _flattenTypeArguments(String? classPrefix, List<TypeDeclaration> args) {
       return _enumName(e.baseName,
           prefix: classPrefix, box: true, suffix: ' *');
     }
-    return _objcTypeForDartType(classPrefix, e).toString();
+    return _objcTypeForDartType(classPrefix, e, forceNullability: true)
```

Comment:
Can you put a comment in the code about why `forceNullability` is true here? (I assume the answer is because "nullable" corresponds to "boxed", and a collection can only have boxed types, but if so that's non-obvious because of the naming so warrants a comment. And if not I don't understand why, so warrants a comment 🙂 )